website/site/layouts/_default/single.html

```diff
@@ -7,22 +7,33 @@
 			<nav class="docs-nav" id="docs-nav">
 				{{ $subjects := where .Site.Pages "Section" "docs" }}
 				{{ range sort $subjects "Params.position" }}
+					{{- if ne .Type "widget" -}}
 					<a href="{{ .Permalink }}" class="nav-link {{ if eq .Params.position $activePage }}active{{ end }}">{{ .Title }}</a>
+					{{- end -}}
 				{{ end }}
 			</nav>
 			<div class="mobile docs-nav">
 				<select class="btn-primary" id="mobile-docs-nav">
 					<option value="" selected="selected">Select A Topic</option>
 					{{ $subjects := where .Site.Pages "Section" "docs" }}
 					{{ range sort $subjects "Params.position" }}
+						{{- if ne .Type "widget" -}}
 						<option value="{{ .Permalink }}" class="nav-link {{ if eq .Params.position $activePage }}active{{ end }}">{{ .Title }}</option>
+						{{- end -}}
 					{{ end }}
 				</select>
 			</div>
 		</aside>
 		<article class="docs-content" id="docs-content">
+			{{- if eq .Title "Widgets V2" -}} <!-- Check if is widgets page, if so, add the partial "widgets" for the widget cloud functionality -->
+				{{ partial "edit-link" . }}
```

Comment:
I think you can move the "edit-link" partial before the conditional, so you don't have to repeat it.